### PR TITLE
Sets `--experimental_link_static_libraries_once=true`

### PR DESCRIPTION
**Pull request by learning-to-play**
This change ensures that no library is linked statically more than once. libtensorflow_framework.so is a dynamic dependency of _pywrap_tensorflow_internal.so. If a library is statically linked to both, Bazel complains and asks for the target to be exported from libtensorflow_framework.so. This change decreases TensorFlow Python package size.

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]

**Review by nitins17 — DISMISSED** [review]

File: check.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INFO:auditwheel.main_repair:Repairing tf_nightly_cpu-2.12.0-cp39-cp39-linux_x86_64.whl
+INFO:auditwheel.wheeltools:Previous filename tags: linux_x86_64
+INFO:auditwheel.wheeltools:New filename tags: manylinux_2_17_x86_64, manylinux2014_x86_64
+INFO:auditwheel.wheeltools:Previous WHEEL info tags: cp39-cp39-linux_x86_64
+INFO:auditwheel.wheeltools:New WHEEL info tags: cp39-cp39-manylinux_2_17_x86_64, cp39-cp39-manylinux2014_x86_64
+INFO:auditwheel.main_repair:
+Fixed-up wheel written to /tf/pkg/tf_nightly_cpu-2.12.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```
Comment (mihaimaruseac) [inline check.txt:7]: This file needs to be removed

File: .bazelrc [hunk 2]
```diff
@@ -142,12 +142,12 @@ build --config=v2
...
 
-# TF now has `cc_shared_library` targets, so it needs the experimental flag
-# TODO(rostam): Remove when `cc_shared_library` is enabled by default
-build --experimental_cc_shared_library
+# Sets no limit to the size of the stdout/stderr files printed to the console.
+build --experimental_ui_max_stdouterr_bytes=-1
 
-# cc_shared_library ensures no library is linked statically more than once.
-build --experimental_link_static_libraries_once=false
+# TF uses `cc_shared_library` targets, so it needs the experimental flag
```
Comment (mihaimaruseac) [inline .bazelrc:148]: Should we keep the TODO here?

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: .bazelrc (hunk 2 above)
Comment (mihaimaruseac) [inline .bazelrc:148]:
```suggestion
# TF uses `cc_shared_library` targets, so it needs the experimental flag
# TODO(rostam): Remove when `cc_shared_library` is enabled by default
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Comment by gbaned** [comment]
Hi @learning-to-play Can you please resolve conflicts? Thank you!

**Comment by learning-to-play** [comment]
Hi @trevor-m, Would you like me to keep this PR? Or will you create a new PR for --`experimental_link_static_libraries_once=true`?

**Comment by trevor-m** [comment]
> Hi @trevor-m, Would you like me to keep this PR? Or will you create a new PR for `--experimental_link_static_libraries_once=true`?

HI @learning-to-play, I think you can close this PR as long as the branch https://github.com/tensorflow/tensorflow-experimental_link_static_libraries_once is kept for my reference

**Pull request closed without merging**